packages/notification-email-sender/src/services/notificationEmailSenderService.ts

```diff
@@ -416,8 +424,20 @@ export function notificationEmailSenderServiceBuilder(
     ) => {
       const purpose = fromPurposeV2(purposeV2Msg);
 
+      const { subject, templateToRetrieve } = match(purpose.versions.length)
+        .with(1, () => ({
+          subject: "Rifiuto delle finalità da parte dell'erogatore",
```

Comment:
```suggestion
          subject: "Rifiuto della finalità da parte dell'erogatore",
```
It seems that this typo was already presents, let's fix it